english/coffee/another-coffee/another-coffee.md

```diff
@@ -10,5 +10,6 @@ You are quite hungry aren't you ?
 [Have another coffee?](../coffee.md)
 
 [Write a novel](../novel/full-length-novel.md)
+[Find stronger coffee!](../starbucks/starbucks-coffee.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.